vital/bindings/c/types/bounding_box.cxx

```diff
@@ -102,7 +120,7 @@ void vital_bounding_box_destroy( vital_bounding_box_t* bbox)
 TYPE vital_bounding_box_ ## NAME( vital_bounding_box_t *bbox )          \
 {                                                                       \
   STANDARD_CATCH(                                                       \
-    "C::bounding_box::" # NAME, 0,                                      \
```

Comment:
@Purg do you have any thoughts on this convention?  Apparently both `C::class::function` and `vital_class_function` are in use in different places in the `STANDARD_CATCH` macro.  I think we should have a follow-on branch to standardize these, but I don't know which way we should go.   Thoughts?  Maybe a hybrid like `[C API] vital_class_function`?